### ISSUE TITLE
parameter-list-wrapping false positive

### ISSUE THREAD

**Issue by allenhumphreys** [pinterest/ktlint#859]
## Observed Behavior

I seem to have found a very strange edge case in `parameter-list-wrapper` rule. It must be an edge case because it took a very long time to find a reasonably minimal reproduction case. Every time I changed something it did not trigger. 

To illustrate, any of the following changes will avoid the false positive:
* Renaming the property `startDateTime` to `property`
* Renaming `TestClassA` to `TestClass`
* Eliminating the line `WORDS c.property = ${"interpolation".length}`

The issue is ktlint triggers the rule and attempt to reformat the line

```kotlin
${FUNCTION_2(TestClassA::startDateTime, descending = true)}
```

to 

```kotlin
${FUNCTION_2(
    TestClassA::startDateTime,
    descending = true
)}
```

## Expected Behavior

I should not encounter a false positive.

## Steps to Reproduce

```kotlin
class Scope {
    fun foo() {
        function("arg1", "arg2") {
            """
            WORDS
            WORDS c.property = ${"interpolation".length}
            ${FUNCTION_2(TestClassA::startDateTime, descending = true)}
            """
        }
    }
}

data class TestClassA(
    val startDateTime: Long
)

class Query
fun function(vararg parameters: String, block: () -> String): Query = Query()
fun FUNCTION_2(property: KProperty1<*, *>, parameterWithDefault: String = "default", descending: Boolean = false): String = ""
```

## Your Environment

* Version of ktlint used: 0.38.1
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): kotlinter
* Version of Gradle used (if applicable): 6.6
* Operating System and version: macOS 10.15.6
